smoke-test.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+set -e
+
+TARBALL_PREFIX=dotnet-sdk-
+VERSION_PREFIX=2.1
+OUTPUT_DIR=bin/x64/Release/
+DOTNET_TARBALL_REL=$(ls ${OUTPUT_DIR}dotnet-sdk-${VERSION_PREFIX}*)
+DOTNET_TARBALL=$(readlink -e ${DOTNET_TARBALL_REL})
+
+projectOutput=false
+keepProjects=false
+dotnetDir=""
+includeWeb=false
+SCRIPT_ROOT="$(pwd)"
```

Comment:
Calling this `SCRIPT_ROOT` seems a bit misleading since it's just the working dir. It looks like `DOTNET_TARBALL_REL` assumes that this script is run with the working dir pointing at the root of a built repo, too--move `SCRIPT_ROOT` to the top, calculate actual script dir, and use it to find the tarball outputs?